### PR TITLE
Feature: add aks tests for notification service

### PR DESCRIPTION
**Pull request by doncem**
### JIRA link (if applicable) ###

[AKS tests for notification service](https://tools.hmcts.net/jira/browse/BPS-1233)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/aat/common/reform-scan/notification-service.yaml [hunk 1]
```diff
@@ -21,6 +28,30 @@ spec:
...
+        serviceAccountName: tests-service-account
+        keyVaults:
+          s2s:
+            resourceGroup: s2s
+            secrets:
+              microservicekey-bulk-scan-processor-tests: S2S_SECRET
+          reform-scan:
+            resourceGroup: reform-scan
+            secrets:
+              notifications-queue-listen-connection-string: NOTIFICATION_QUEUE_CONN_STRING_READ
```
Comment (mdayican) [inline k8s/aat/common/reform-scan/notification-service.yaml:42]: staging?

Comment (doncem) [reply k8s/aat/common/reform-scan/notification-service.yaml:42]: staging is for jenkins pipelines as it deploys to staging pods. flux deploys to "real" stuff so - not staging. unless i'm getting it wrong

File: k8s/aat/common/reform-scan/notification-service.yaml [hunk 2]
```diff
@@ -21,6 +28,30 @@ spec:
       replicas: 2
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/reform-scan/notification-service:prod-8cdadccd
+      testsConfig:
+        memoryLimits: "3072Mi"
+        serviceAccountName: tests-service-account
+        keyVaults:
+          s2s:
+            resourceGroup: s2s
+            secrets:
```
Comment (nitinprabhu) [inline k8s/aat/common/reform-scan/notification-service.yaml:37]: should be in bulk scan key vault

Comment (doncem) [reply k8s/aat/common/reform-scan/notification-service.yaml:37]: i'll swap with that then ;)

**Pull request merged**